src/confd/yang/infix-system@2024-09-13.yang

```diff
@@ -242,7 +242,10 @@ module infix-system {
         base shell-type;
       }
       default false;
-      description "Set UNIX login shell for user, default: none (security)";
+      description "Set UNIX login shell for user. For security reasons avoid
+                   shells bash and sh for non-admin users. This since this
+                   may open the system for local security issues.
+                   default: none (security)";
```

Comment:
Suggest double line breaks between first and second sentence for better display in `help shell` output.

Same for default value.